Source/CardViewController.swift

```diff
@@ -196,11 +196,15 @@ public class CardViewController: UIViewController,
                                        phone: billingDetailsPhone)
         if let checkoutApiClientUnwrap = checkoutApiClient {
```

Comment:
optional just naming : but you can instead `if let checkoutApiClient = checkoutApiClient {` adding Unwrap is not required.